.github/workflows/build-validation.yml

```diff
@@ -0,0 +1,42 @@
+name: build
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+    paths:
+    - '**.cs'
+    - '**.csproj'
+    - '**.cshtml'
+    - '**.css'
+
+env:
+  DOTNET_VERSION: '6.0.101' # The .NET SDK version to use
```

Comment:
isn't there a way to always use the latest .NET 6 SDK?

Reply:
hmmmmmmmmmmm good question... yes